Source/WebKit/Shared/ApplePay/ApplePayPaymentSetupFeatures.serialization.in

```diff
@@ -26,7 +26,7 @@ headers: <pal/cocoa/PassKitSoftLink.h>
 
 header: "ApplePayPaymentSetupFeaturesWebKit.h"
 [CustomHeader] class WebKit::PaymentSetupFeatures {
-    [SecureCodingAllowed=[NSArray.class, PAL::getPKPaymentSetupFeatureClassSingleton()]] RetainPtr<NSArray> m_platformFeatures;
+    Vector<RetainPtr<PKPaymentSetupFeature>> serializableFeatures();
```

Comment:
hmmm, unless I'm misunderstanding, we would need to adopt this in the IPC receiver right? For example, `WebPaymentCoordinatorProxy::platformBeginApplePaySetup()` is still doing `[request setPaymentSetupFeatures:features.protectedPlatformFeatures().get()];`

Reply:
The holding object in each process still keeps the NSArray. We just convert at the point where we serialize and then re-materialize in the receiving process. This means we shouldn’t need any wider adoption.